src/NzbDrone.Core/Notifications/Email/EmailSettings.cs

```diff
@@ -55,16 +50,16 @@ public EmailSettings()
         [FieldDefinition(4, Label = "Password", Type = FieldType.Password, Privacy = PrivacyLevel.Password)]
         public string Password { get; set; }
 
-        [FieldDefinition(5, Label = "From Address")]
+        [FieldDefinition(5, Label = "From Address", Placeholder = "example@email.com")]
```

Comment:
Add a pretty format example `PrettyName-Radarr <somemail@gmail.com>`

Reply:
That won't send though... it's not an SMTP address in that format

Reply:
Works fine here
https://github.com/Radarr/Radarr/commit/f120b84c43a9ced3a225b6402baed433c546fc4e

Reply:
This isn't done yet, but @Qstick up to you if you want it in pretty format or just email address (my preference is just email personally)